rollup.config.js

```diff
@@ -32,7 +34,13 @@ function createESMConfig(input, output) {
     external,
     plugins: [
       resolve({ extensions }),
-      typescript(),
+      esbuild({
+        minify: process.env.NODE_ENV === 'production',
+        target: 'esnext',
+        jsxFactory: 'React.createElement',
+        jsxFragment: 'React.Fragment',
+        tsconfig: path.resolve('./tsconfig.json'),
```

Comment:
I thought esbuild just strips type information. Do they follow some tsconfigs? (Out of curiosity...

Reply:
https://esbuild.github.io/content-types/#tsconfig-json

Reply:
> Only certain tsconfig.json fields are respected

I see. Thanks.